.github/workflows/flake8.yaml

```diff
@@ -21,6 +21,9 @@ jobs:
       with:
         python-version: 3
 
+    - name: Install krb5-config deps
+      run: sudo apt-get install -y libkrb5-dev gcc
```

Comment:
I don't know why update and upgrade will be an issue from before, maybe try something like:
"
run: sudo apt-get update -y \
           && sudo apt-get upgrade -y \
           && sudo apt-get install -y \
           libkrb5-dev \
           gcc
"
And if it doesn't work then we keep this. If it is not too much effort otherwise we keep this.

Reply:
It worked like that.  Maybe the backslash I tried using for continuation was the issue before.